src/encoded/tests/test_search.py

```diff
@@ -238,8 +238,8 @@ def test_set_filters():
             }
         }
     }
-    result = {'filters': []}
-    used_filters = set_filters(request, query, result)
+
```

Comment:
Do we actually need to repeat all these snovault/search tests in encoded?  It seems like we should just append the ones that are used in encoded specific search sub modules.